test/common.js

```diff
@@ -453,3 +453,9 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+exports.busyLoop = function busyLoop(time) {
+  var startTime = new Date().getTime();
+  var stopTime =  startTime + time;
+  while (new Date().getTime() < stopTime);
```

Comment:
`Timer.now()` has a fighting chance of fitting in an SMI (a tagged integer), the return value of `Date.now()` is always a heap-allocated double.